Cargo.toml

```diff
@@ -100,6 +100,7 @@ tracing-subscriber = { version = "0.3", features = ["registry", "env-filter", "j
 url = "2.2"
 x509-parser = { version = "0.16", default-features = false }
 backoff = "0.4.0"
+tracing-core = "0.1.32"
```

Comment:
Is this compatible with OTLP ?

Reply:
Its just the `tracing` library, so its pluggable. Right now we don't have code to do it, but we could.

Even better, when we do we could easily choose to send JUST access logs (or send them to a different place, handle them differently, whatever)